component/otelcol/extension/jaegerremotesampling/jaegerremotesampling.go

```diff
@@ -0,0 +1,88 @@
+package jaegerremotesampling
+
+import (
+	"time"
+
+	"github.com/grafana/agent/component"
+	"github.com/grafana/agent/component/otelcol"
+	"github.com/grafana/agent/component/otelcol/extension"
+	"github.com/grafana/agent/pkg/river"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/extension/jaegerremotesampling"
+	otelcomponent "go.opentelemetry.io/collector/component"
+	otelconfig "go.opentelemetry.io/collector/config"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name: "otelcol.extension.jaegerremotesampling",
```

Comment:
We typically use `snake_case` for component names unless the thing we're referring to is a multi-word identifier (like `podlogs` for PodLogs). 

Can we change this `otelcol.extension.jaeger_remote_sampling`?